netbout-web/src/test/java/com/netbout/rest/bout/TkInviteTest.java

```diff
@@ -88,4 +91,46 @@ public void invitesUserByEmail() throws Exception {
             throw ex;
         }
     }
+
+    /**
+     * TkInvite should throw an RsFailure when invite alias is too long.
+     * @throws Exception If there is some problem inside
+     */
+    @Test (expected = RsFailure.class)
+    public void inviteWithTooLongAlias() throws Exception {
+        final MkBase base = new MkBase();
+        final String urn = "urn:test:1";
+        final User user = base.user(new URN(urn));
+        final String name = "12345678901234567890123456789012345678901234"
+            + "5678901234567890123456789012345678901234567"
+            + "890123456789012345678901234567890";
+        user.aliases().add(name);
+        final Alias alias = user.aliases().iterate().iterator().next();
+        final Bout bout = alias.inbox().bout(alias.inbox().start());
+        try {
+            new TkInvite(base).act(
+                new RqWithHeader(
+                    new RqWithAuth(
+                        urn,
+                        new RqFake(
+                            RqMethod.POST,
+                            String.format(
+                                "/b/%d/invite",
+                                bout.number()
+                            ),
+                            "name=" + name
```

Comment:
@jhyle Same here, let's avoid string concat. In this case, you can use `String.format("name=%s", name)`